boilerplate/update

```diff
@@ -27,7 +27,7 @@ else
     BOILERPLATE_GIT_REPO=git@gitlab.cee.redhat.com:service/boilerplate.git
   fi
   if [ -z "$BOILERPLATE_GIT_CLONE" ]; then
-    BOILERPLATE_GIT_CLONE="git clone $BOILERPLATE_GIT_REPO"
+    BOILERPLATE_GIT_CLONE="git clone --branch=golangci-lint $BOILERPLATE_GIT_REPO"
```

Comment:
Don't forget to remove this.

(The point of having the `BOILERPLATE_GIT_CLONE` variable was so you could set this up in your env and not have to hardcode it into the repo temporarily.)